Gems/Atom/RHI/Code/Include/Atom/RHI/FrameGraphExecuteContext.h

```diff
@@ -48,6 +49,9 @@ namespace AZ
 
             /// Returns the command list associated with the index in the batch.
             CommandList* GetCommandList() const;
+
+            /// Returns the submit range for this context.
+            const CommandList::SubmitRange& GetSubmitRange() const { return m_descriptor.m_submitRange; }
```

Comment:
It is a bit weird to see a bunch of getters and setters and only this one implemented in the headers. Lets move the implementation to the cpp to match the other functions here.

Reply:
This adds more cruft to the code. It would be nice if you can move the getter/setters from cpp to the header than to be consistent. Thanks.

Reply:
I moved the rest of the accessors to the header.  Honestly it seems like we should just provide a single accessor for the Descriptor at this point, but that's an exercise for another PR.